lib/screens/home/components/recomend_products.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+
+import '../../../constants.dart';
+
+class RecomendsProducts extends StatelessWidget {
+  const RecomendsProducts({
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      scrollDirection: Axis.horizontal,
+      child: Row(
+        children: <Widget>[
+          RecomendProductCard(
```

Comment:
Nice work, but maybe better if you use **map or builder** and work with objects to optimize your code :)